.github/workflows/automerge.yaml

```diff
@@ -34,16 +30,10 @@ jobs:
           direction: last
 
       - name: Fail automerge if PRT was not initiated
-        if: steps.find-prt-comment.outputs.comment-body == ""
+        if: steps.find-prt-comment.outputs.comment-body == ''
         run: |
           echo "::error PRT comment not added the PR"
 
-      - name: Wait for PRT checks to get initiated
```

Comment:
Isnt this needed anymore ?

Reply:
we are not going to trigger the event using the comment hence, the user can take time and once PRT starts they can trigger just reloading the label

Reply:
But thats very manual process. If someone forgets to reload the label, your PR will be auto-merged without PRT running/passing.

Reply:
if in that case we can add 5 mins delay for every occurrence

Reply:
@jyejare if you are okay we can do this then?

Reply:
I removed the issue_comment trigger as this only works in the default branch master, we don't want to run it here on the master. To avoid that I'm removing that.   see here  https://github.com/SatelliteQE/robottelo/actions/runs/3740710868

Reply:
Sure! For now we can merge as is.

Later, we can find some GHA that waits for some GHA to start or we can build one if needed :)